src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SupplementaryEvidenceTest.java

```diff
@@ -40,12 +43,23 @@
 
     private CaseDetails caseDetails;
 
+    private String dmUrl;
+
+    @Autowired
+    private DocumentManagementUploadService dmUploadService;
+
     @Before
     public void setup() {
         String caseData = SampleData.fileContentAsString("envelopes/new-envelope.json");
         Envelope newEnvelope = EnvelopeParser.parse(caseData);
 
         caseDetails = ccdCaseCreator.createCase(newEnvelope);
+
+        UploadResponse uploadResponse = dmUploadService.uploadToDmStore(
+            "supplementary-evidence.pdf",
+            "documents/supplementary-evidence.pdf"
```

Comment:
could it just take a path and extract the file name from it?

Reply:
The file name will be used to display on the case, so it can be different from the path. I've now renamed the field name to "displayName"